### PR TITLE
Crontab Cheat Sheet: Add JSON file

### PR DESCRIPTION
**Pull request by rasikapohankar**
###### Description of changes

Add JSON file to change the Crontab Goodie to a cheat sheet.

###### People to notify:

@GuiltyDolphin @moollaza 

---

Instant Answer Page: https://duck.co/ia/view/crontab_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @nkorth

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/crontab.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
+{
+    "id": "crontab_cheat_sheet",
+    "name": "Crontab",
+    "description": "Examples explaining crontab lines",
+    "metadata": {
+        "sourceName": "CronHowto",
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/crontab.json]: Perhaps `Ubuntu Documentation` here?

Comment (rasikapohankar) [inline share/goodie/cheat_sheets/json/crontab.json]: @gaulrobe :+1: I had used `Ubuntu Documentation Community Help Wiki` earlier and then removed it :smile:

**Comment by edgesince84** [comment]
thanks @rasikapohankar

**Comment by tagawa** [comment]
This is now installed on beta but it looks like the original IA is still triggering. @moollaza, do we need to disable the previous Goodie first?

**Comment by rasikapohankar** [comment]
Thanks @tagawa , #3368

**Comment by tagawa** [comment]
Cool, with help from @moollaza I updated the IA page, redeployed and this cheat sheet now works on beta for testing: https://beta.duckduckgo.com/?q=crontab+cheat+sheet&ia=cheatsheet
If there are no further comments we can merge this soon.

**Comment by GuiltyDolphin** [comment]
@rasikapohankar @tagawa Looks good to me - jus' waiting for maintainer feedback then we should be good to go!

**Comment by tagawa** [comment]
Hi @knorth. Are you happy with the changes here?

**Comment by tagawa** [comment]
I think the maintainer is busy but the beta implementation looks good to me so let's merge.
Thank you very much for this @rasikapohankar! When it goes live I'll then get the existing IA removed so this cheat sheet can appear.

**Pull request merged**